src/libmongoc/tests/unified/operation.c

```diff
@@ -135,19 +141,36 @@ operation_list_databases (test_t *test,
    bool ret = false;
    mongoc_client_t *client = NULL;
    mongoc_cursor_t *cursor = NULL;
+   bson_parser_t *parser = NULL;
+   bson_t *opts = NULL;
+
+   parser = bson_parser_new ();
+   bson_parser_allow_extra (parser, true);
+
+   opts = bson_copy (bson_parser_get_extra (parser));
```

Comment:
`parser` is not parsing any input BSON. This can be simplified to `opts = bson_new ()`.

Reply:
Ah gotcha; no need to add extra parsing if there wasn't any in the first place! Simplified to `opts = bson_new ()` and removed the `bson_parser_allow_extra`.

Reply:
Was this change not pushed? I still see `parser = bson_parser_new ();` and `bson_parser_allow_extra (parser, true);` here.